groundcontrol/admin-bundle.tasks.js

```diff
@@ -51,19 +50,21 @@ adminBundle.tasks.scripts = createScriptsTask({
         './node_modules/jquery.typewatch/jquery.typewatch.js',
         './node_modules/ckeditor/ckeditor.js',
         './node_modules/ckeditor/adapters/jquery.js',
-        adminBundle.config.srcPath + 'js/**/*.js'
+        `${adminBundle.config.srcPath}js/**/*.js`
     ],
-    dest: adminBundle.config.distPath + 'js',
+    dest: `${adminBundle.config.distPath}js`,
     filename: 'admin-bundle.min.js'
 });
 
 adminBundle.tasks.bundle = createBundleTask({
     config: {
-        entry: adminBundle.config.srcPath + 'jsnext/app.js',
+        entry: `${adminBundle.config.srcPath}jsnext/app.js`,
         output: {
-            filename: adminBundle.config.distPath + 'js/admin-bundle.next.js',
+            filename: `admin-bundle.next.js`,
+            path: path.resolve(__dirname, '../src/Kunstmaan/AdminBundle/Resources/public/js'),
```

Comment:
Can you use the adminBundle.config here? As all other paths are in the config object